cluster/environments/azure-simple/variables.tf

```diff
@@ -15,6 +15,11 @@ variable "agent_vm_count" {
     default = "3"
 }
 
+variable "agent_vm_size" {
```

Comment:
@samiyaakhtar is this a merge from master?

Reply:
Yes it was due to a merge, I will have to undo it